measures/iama.yaml

```diff
@@ -0,0 +1,130 @@
+systemcard_path: .assessments[]             # append iama.yaml to 'assessments' list in system card 
```

Comment:
I think we should add `schema_version` as well (referencing the schema version)

Reply:
do you mean systemcard_version?